src/dev-app/cdk-experimental-listbox/cdk-listbox-demo.html

```diff
@@ -0,0 +1,10 @@
+<ul cdkListbox class="example-listbox" [multiple]="multiSelectable" [useActiveDescendant]="activeDescendant">
+  <li class="example-option" cdkOption>Apple</li>
```

Comment:
Is this class a noop right now?

Reply:
Ah yes I forgot to remove it after I changed the way I did the styling.